ui/raidboss/data/05-shb/trial/ruby_weapon.js

```diff
@@ -261,14 +261,14 @@
         'Mark II Magitek Comet': 'Comète magitek II',
         'Magitek Meteor': 'Météore magitek',
         '(?<! )Magitek Comet': 'Comète magitek',
-        'Landing': 'Atterrissage rapide',
+        'Landing': 'Chute de Météore',
         'Liquefaction': 'Sables mouvants',
         '(?<! )Homing Lasers': 'Lasers autoguidés',
         'High-Powered Homing Lasers': 'Lasers autoguidés surpuissants',
         'Helicoclaw': 'Héliogriffes',
         'Flexiclaw': 'Flexigriffes',
         'Dalamud Impact': 'Impact de Dalamud',
-        'Burst': 'Explosion',
+        'Burst': 'Éclats cosmique',
```

Comment:
Don't touch plz.
https://fr.fflogs.com/reports/1nf9kAPbMhRt6vLC#fight=12&type=casts&hostility=1&by=ability (id=19186) english
https://fr.fflogs.com/reports/KyzPwcJDHvxdY1ga#fight=6&type=casts&by=ability&hostility=1 (id=19186) french

Reply:
the linein the timeline file says `1115.9 "Burst" sync /:Comet:4AB4:/` where `4AB4` is `19124` which is based on Godbert `Explosion` in English

Reply:
I will therefore do the content myself because godbert and saint coinach have some translations error that I noticed. FFlogs read us logs, i think is better.

Reply:
Ok, there is a difference between the names normal and extreme. I had not noticed it, the correction is welcome. Thank you.